backend/typescript/rest/entityRoutes.ts

```diff
@@ -28,6 +28,7 @@ const entityService: IEntityService = new EntityService(fileStorageService);
 /* Create entity */
```

Comment:
@dhmi19 I think we should just delete this file + along with all starter code legacy

Reply:
yeah do we need UserRoutes either?

Reply:
jk, we need some userRoutes. But I think we can delete the create user and delete user endpoints?

Reply:
To clarify, I think we need to keep these routes since we are keeping image uploads